docs/docs/reference/dropped-features/wildcard-init.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: doc-page
+title: "Dropped: wildcard initializer"
+---
+
+The syntax
+```scala
+  var x: A = _
+```
+that was used to indicate an uninitialized field has been dropped.
+At its place there is a special value `notInitialized` in the `scala.compiletime` package. To get an uninitialized field, you now write
```

Comment:
Minor bikeshed: since we call them uninitialized fields, why not call this method `uninitialized` too?